File: {PWB}/If-Else Programming/001. Positive Or Negative/Java/solution.java
import java.util.Scanner;
import java.io.*;

public class Main
{
	public static void main(String[] args) 
    {
        int number;
        Scanner scan = new Scanner(System.in);
        System.out.print("Enter The Number You Want to Check: ");
        number = scan.nextInt();
        if(number > 0)
        {
            System.out.println("Positive");
        }
        else if(number < 0)
        {
            System.out.println("Negative");
        }
        else
        {
            System.out.println("Neutral");
        }
    }
}

File: {PWB}/If-Else Programming/001. Positive Or Negative/C/solution.c
#include<stdio.h>

int main() {
    int a;
    printf("Enter The Number You Want To Check :\n");
    scanf("%d",&a);
 
    if(a<0) {
      printf("Negative\n");
    }
    else if(a>0) {
      printf("Positive\n");
    } 
    else {
     printf("Neutral");
    }
    return 0;

}
